src/ds++/test/tstField_Traits.cc

```diff
@@ -20,27 +21,27 @@ using namespace rtt_dsxx;
 
 void tstFT(UnitTest &ut) {
   if (Field_Traits<complex<double>>::zero() == 0.0)
-    ut.passes("complex zero good");
+    PASSMSG("complex zero good");
   else
-    ut.failure("complex zero NOT good");
+    FAILMSG("complex zero NOT good");
   if (Field_Traits<complex<double>>::one() == 1.0)
-    ut.passes("complex zero good");
+    PASSMSG("complex zero good");
   else
-    ut.failure("complex zero NOT good");
+    FAILMSG("complex zero NOT good");
   double const x = 3.7;
-  if (value(x) == 3.7)
-    ut.passes("complex zero good");
+  if (rtt_dsxx::soft_equiv(value(x), 3.7))
+    PASSMSG("complex zero good");
   else
-    ut.failure("complex zero NOT good");
+    FAILMSG("complex zero NOT good");
 
-  if (Field_Traits<double const>::zero() == 0.0)
-    ut.passes("double zero good");
+  if (rtt_dsxx::soft_equiv(Field_Traits<double const>::zero(), 0.0))
```

Comment:
I think this zero could use the tightest tolerance available, maybe `numeric_limits<double>::min()` and the same thing for the check on line 42?